sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/ChangeFeedProcessor.java

```diff
@@ -26,13 +26,13 @@
  * <p>
  * {@code
  * ChangeFeedProcessor changeFeedProcessor = ChangeFeedProcessor.Builder()
- *     .hostName(hostName)
- *     .feedContainer(feedContainer)
- *     .leaseContainer(leaseContainer)
- *     .handleChanges(docs -> {
+ *     .setHostName(setHostName)
+ *     .setFeedContainer(setFeedContainer)
+ *     .setLeaseContainer(setLeaseContainer)
+ *     .setHandleChanges(docs -> {
```

Comment:
It would be good to file a follow-up work item to extract these code samples out into Java-compilable code snippets. I can walk you through how to do this as appropriate.

Reply:
#5626